samples/app/build.gradle

```diff
@@ -85,4 +85,7 @@ dependencies {
     androidTestCompile rootProject.spoon
     androidTestCompile rootProject.espresso
     androidTestCompile rootProject.androidTestingSupport
+    compile('com.crashlytics.sdk.android:crashlytics:2.5.5@aar') {
```

Comment:
Can we remove changes to the Sample application? The goal is to just demonstrate the usages of TwitterKit.